src/main/java/com/socrata/datasync/HttpUtility.java

```diff
@@ -36,19 +36,30 @@
     private String authHeader;
     private String appToken;
     private boolean authRequired = false;
+    private final int maxRetries;
+    private final double retryDelayFactor;
+
     private static final String datasyncVersionHeader = "X-Socrata-DataSync-Version";
     private static final String appHeader = "X-App-Token";
     private static String userAgent = "datasync";
 
 
     public HttpUtility() { this(null, false); }
 
+
+    public HttpUtility(UserPreferences userPrefs, boolean useAuth) {
+        this(userPrefs, useAuth, 5, 3.5);
```

Comment:
Should this 5 & 3.5 be maxRetries & retryDelayFactor?  They're new and set but never used.